infra/azure/terraform/iam-config/param.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "prow-identity-credential",
+    "issuer": "https://container.googleapis.com/v1/projects/k8s-infra-prow-build/locations/us-central1/clusters/prow-build",
+    "subject": "system:serviceaccount:test-pods:prowjob-default-sa",
+    "audiences": [
+        "api://AzureADTokenExchange"
+    ]
+}
```

Comment:
Nit: add newlines to the end of all these files that have the :heavy_minus_sign: sign